deployments/dev/images/secondary/environment.yml

```diff
@@ -7,69 +7,69 @@ channels:
 dependencies:
 - python==3.11.*
 - git==2.39.1
+- jupyterhub==5.0.0
 - jupyter-resource-usage==1.0.0
 - jupyterlab==4.0.11
 - jupyterlab-favorites==3.0.0
+# Install myst for generating dashboards
+- jupyterlab-myst==2.4.2
 - jupyterlab_server==2.23.0
 - jupyterlab_widgets==3.0.8
 - jupyter_server==2.7.0
 - nbgitpuller==1.2.1
 - notebook==7.0.7
 - folium==0.14.0
-- h5netcdf==1.0.2
+  #- h5netcdf==1.0.2
```

Comment:
Feel free to delete old entries rather than comment them out, unless you're not certain and need a recent reminder. We can always look back into git history if needed. Sometimes I leave larger commented stanzas in as kind of a template.

Reply:
Thanks Ryan! It makes sense. I have removed the entries from the yml file.

  #- h5netcdf==1.0.2
  #- mdit-py-plugins==0.4.0
  #- jupysql==0.8.0
  #- scikit-learn==1.2.2
  #- scipy==1.10.1
# Spring 2024 table demos
#- lxml==5.1.0
# Spring 2024 data 100
#- pytorch==2.1.2
#- cpuonly==2.0
#- transformers==4.37.1
  #- statsmodels==0.14.0
  #- tensorflow-cpu==2.12.1
  #- sqlalchemy==2.0.16
  #- mlxtend==0.23.0
  #- lifelines==0.27.8
  #- pytest-notebook==0.10.0
  #- gh-scoped-creds==4.1
  #- ydata-profiling==4.8.3
  #- otter-grader==5.5.0
  #- python-duckdb==1.0.0
  #- pip
  #- pip:
  # - -r infra-requirements.txt
  # disable until fixed (probably this:  https://github.com/jupyterlab/jupyter-collaboration/issues/162)
  # - jupyter_collaboration==1.0.1
  #- jupyterhub==4.1.5
  # - nbconvert[webpdf]
  #  - pyppeteer==2.0.0
  #- pytest-notebook==0.8.1
  #- gh-scoped-creds==4.1
  # - git+https://github.com/shaneknapp/python-popularity-contest.git@add-error-handling
  #- ydata-profiling==4.6.4
  #- otter-grader==5.4.0
  #- duckdb==0.10.1
  #  - duckdb_engine==0.11.2